kafka-client/src/test/java/dev/responsive/kafka/internal/db/AbstractCassandraKVTableIntegrationTest.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2024 Responsive Computing, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package dev.responsive.kafka.internal.db;
+
+public class AbstractCassandraKVTableIntegrationTest {
```

Comment:
used?

Reply:
nope this is TODO -(going to combine some of the common KV vs Fact table tests, especially the ttl unit tests)

Reply:
if possible, I'd prefer we used static helper methods to abstract classes for tests 🤷

Reply:
It's not about the static helper methods it's about the test themselves. We want to run largely the same set of tests for both of these (same insert semantics, same ttl semantics, etc) so I was going to extract all the common tests into this class. 

If you'd prefer the @Parametrize approach I can take a look at that, but since not all the tests are going to be the same for the kv vs fact tables, I thought the abstract test format would be cleaner. Does that make sense?

(That said, ideally we can share the same tests for the Mongo table as well, in which case another approach might be easier in the end 🤷‍♀️ )